lib/mayaUsd/ufe/UsdUIInfoHandler.cpp

```diff
@@ -68,6 +69,46 @@ void addMetadataCount(
         addMetadataStrings(refs.size(), tooltip, needComma, singular, plural);
     }
 }
+
+std::vector<double>& getInvisibleColor()
+{
+    static std::vector<double> rgb;
+    return rgb;
+}
+
+void updateInvisibleColor()
+{
+    MDoubleArray color;
+    MGlobal::executeCommand("displayRGBColor -q \"outlinerInvisibleColor\"", color);
+
+    // Note: support future possible alpha channel for invisible color.
+    if (color.length() >= 3) {
+        auto& rgb = getInvisibleColor();
+        rgb.resize(color.length());
+        color.get(rgb.data());
+    }
+}
```

Comment:
There is no need for this to be a vector. If you look at the command docs for `displayRGBColor` you need to add the `-alpha` flag to the query to get 4 floats. So I would suggest making this an std::array with a fixed size of 3.

Reply:
I used to the size to detect that the array was initialized. I can use a separate variable for it.